Assets/Scripts/SafeZoneLogic.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SafeZoneLogic : MonoBehaviour
+{
+    [SerializeField]
+    private GameObject player;
+    [SerializeField]
+    private int radius;
+    private GameObject[] pickables;
+    private int returnedPickables;
+
+    void Start()
+    {
+        pickables = GameObject.FindGameObjectsWithTag("PickableObject");
+    }
+
+    void Update()
+    {
+        if (player.GetComponent<HoldObject>().get_PickedUpObject() == null)
+        {
+            if (IsTargetInRange(player, radius) == true)
+            {
```

Comment:
```suggestion
        if (player.GetComponent<HoldObject>().get_PickedUpObject() == null && IsTargetInRange(player, radius))
        {
```

Reply:
Also is there a specific reason you used `IsTargetInRange(player, radius) == true`?